### ISSUE TITLE
PHP configuration misleading

### ISSUE THREAD

**Issue by tablatronix** [webketje/GSconfig-UI#4]
This section is misleading, none of these settings have anything to do with php itself.

**Comment by webketje** [comment]
I agree. I'm not a die-hard back-end guy, do you have any suggestions for how to rename it? Should some of the settings in this section be moved to somewhere else?

**Comment by tablatronix** [comment]
Maybe `advanced`
